tests/e2e-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+echo "starting rekor"
+docker compose up -d --build
+
+echo "waiting up to 2 minutes for system to start"
+count=0
+until [ $(docker inspect rekor-tiles-rekor-1 | jq -r .[0].State.Health.Status) == "healthy" ];
+do
+    if [ $count -eq 24 ]; then
+        echo "timeout reached"
+        exit 1
+    else
+        echo -n "."
+        sleep 5
+        let 'count+=1'
+    fi
+done
```

Comment:
I think we can replace this with 

```
docker compose up --wait --wait-timeout 60
```

And with the compsoe.yml changes in #142 it can be ready much more quickly.

Reply:
`--wait` is a compose v2 feature I suppose?